IOMC/EventVertexGenerators/src/HLLHCEvtVtxGenerator.cc

```diff
@@ -197,7 +197,7 @@ double HLLHCEvtVtxGenerator::integrandCC(double x, double z, double ct) const {
         std::exp(-ct * ct * ct * ct / sigs4 - z * z * z * z * cos2 * cos2 / sigs4 - 6 * ct * ct * z * z * cos2 / sigs4 -
                  sin2 / (4 * k2 * sigmax2) *
                      (2 + 4 * k2 * z * z - std::cos(2 * k * (z - ct)) - std::cos(2 * k * (z + ct)) -
-                      8 * k * s * std::cos(k * ct) * std::sin(k * z) - 4 * cosks * cosks * sinct * sinct)) /
+                      8 * k * CLHEP::s * std::cos(k * ct) * std::sin(k * z) - 4 * cosks * cosks * sinct * sinct)) /
         std::sqrt(1 + z * z / (betx * betx)) / std::sqrt(1 + z * z / (bets * bets));
```

Comment:
```suggestion
        std::sqrt((1 + z * z / (betx * betx)) / (1 + z * z / (bets * bets)));
```
could avoid one square root...

Reply:
accepting here only because it's on a non-working branch of the code; otherwise it would trigger numerical differences.